### ISSUE TITLE
Contrast on special text in night mode

### ISSUE THREAD

**Issue by Link2Twenty** [forem/forem#2152]
The new spring banner is difficult to read in night mode. Is there a way to automate contrast checks? Or should we just be mindful when making special styles?

![image](https://user-images.githubusercontent.com/3534427/54747199-2356a700-4bc6-11e9-8e73-7160c69c656c.png)

**Comment by edisonywh** [comment]
Not exactly automated but someone just [posted this on dev.to](https://dev.to/iriskatastic/color-review-is-a-tool-to-test-color-contrast-4m55), should be a good manual tool until we can find an automated one

**Comment by aspittel** [comment]
@Link2Twenty thanks for reporting this, and @edisonywh thanks for the resource! This will definitely be something important to check out as we move forward with theming.

**Comment by nickytonline** [comment]
@Link2Twenty, is this still an issue. Only asking as the frontend has undergone massive changes since this issue was created.

**Comment by Link2Twenty** [comment]
How are these special side panels made?
I believe that example was caused by human 'error' making it look nice in one theme but not others.
I'm not sure if there's anything we can do about that?

**Comment by cmgorton** [comment]
Thanks for bring this up. I think this issue will now be encompassed in the broader accessibility issue Marcy has opened #11504. If we have any more comments or suggestions we can leave them in that issue. Since that is the case I am going to close this issue.
